docs/brushes/TilesBrush.md

```diff
@@ -0,0 +1,62 @@
+---
+title: TilesBrush
+author: Sergio0694
+description: A composition brush that that displays a tiled image.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, brush, backdrop, blur, win2d, composition
+---
+
+# TilesBrush
+
+The [TilesBrush](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.media.tilesbrush) is a [Brush](https://docs.microsoft.com/uwp/api/windows.ui.xaml.media.brush) that that displays a tiled image.
+
+> [!div class="nextstepaction"]
+> [Try it in the sample app](uwpct://Brushes?sample=TilesBrush)
+
+## Syntax
+
+```xaml
+<Border BorderBrush="Black" BorderThickness="1" VerticalAlignment="Center" HorizontalAlignment="Center" Width="400" Height="400">
+  <Border.Background>
+    <brushes:TilesBrush TextureUri="ms-appx:///Assets/BrushAssets/TileTexture.png"/>
+  </Border.Background>
+</Border>
+```
+
+## Example Image
+
+![Tiles brush](../resources/images/Brushes/TilesBrush.jpg "Tiles brush")
+
+## Properties
+
+| Property | Type | Description |
+| -- | -- | -- |
+| TextureUri | Uri | The `Uri` for the texture to use. |
+
+## Code behind support
+
+This brush can be also be built from code behind through the `PipelineBuilder` class.
```

Comment:
Maybe call out a bit more about how this can be used to customize the scaling and other aspects of the tiling (and link to the page itself directly here as well)?

Reply:
Added the `DpiMode` property to the `TilesBrush` in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3112/commits/ade1e04a1d5696c37f2da8d6b68398376d2a92fc so that it's possible to change that without having to use code behind and/or write a custom brush. Updated the docs about that new property in https://github.com/MicrosoftDocs/WindowsCommunityToolkitDocs/pull/294/commits/b02307642834aa2a52d85664e5dfacc4ce891657.